packages/google_maps_flutter/google_maps_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Adds support for patterns in polylines on iOS.
```

Comment:
do we need this change in android?

Reply:
I hadn't done this, and was getting some error from an automated test about README. So I ran the tool instead of doing the changelog manually, and that added this here

Reply:
You can revert this; we'll override the changelog check once it's ready to land. The tool errs on the side of adding changelogs for things so that wo don't just forget them, but the current `google_maps_flutter_android` change is uninteresting for clients, so shouldn't have an entry here.

Reply:
This still needs to be reverted.